pkg/metrics/controller_metrics.go

```diff
@@ -42,19 +44,21 @@ var (
 	// fleetViews are metric views associated with Fleets
 	fleetViews = append([]string{fleetReplicaCountName, gameServersCountName, gameServersTotalName, gameServerStateDurationName}, fleetAutoscalerViews...)
```

Comment:
We'll want to add `gameServerPlayerConnectedTotal` and `gameServerPlayerCapacityTotal` to `fleetViews` - so that the system knows to reset them when Fleets get deleted,.

Reply:
Good catch. I just added that.